components/utils.py

```diff
@@ -118,16 +128,17 @@ def sudo(command):
     run(command)
 
 
-def error_exit(message):
-    logger.info(message)
-    sys.exit(1)
+def error_exit(message, returncode=1):
+    logger.error(message)
+    sys.exit(returncode)
 
 
-def create_dir(dir):
-    if os.path.isdir(dir):
+def create_dir(directory):
+    if os.path.isdir(directory):
         return
-    logger.info('Creating Directory: {0}'.format(dir))
-    sudo(['mkdir', '-p', dir])
+    else:
```

Comment:
I added it when I was tweaking the name to make the logic flow more explicit- either the directory exists OR we create it.

Also means that we don't accidentally try to create it if we decide that there needs to be an extra line of logging, it won't cause problems if whoever implements it brainos and deletes the return.